packages/cli/src/commands/database/key.ts

```diff
@@ -0,0 +1,96 @@
+import { logtoConfigGuards, LogtoConfigKey, logtoConfigKeys } from '@logto/schemas';
+import chalk from 'chalk';
+import { CommandModule } from 'yargs';
+
+import { createPoolFromConfig } from '../../database';
+import { getRowsByKeys, updateValueByKey } from '../../queries/logto-config';
+import { deduplicate, log } from '../../utilities';
+
+const validKeysDisplay = chalk.green(logtoConfigKeys.join(', '));
+
+type ValidateKeysFunction = {
+  (keys: string[]): asserts keys is LogtoConfigKey[];
+  (key: string): asserts key is LogtoConfigKey;
+};
+
+const validateKeys: ValidateKeysFunction = (keys) => {
+  const invalidKey = (Array.isArray(keys) ? keys : [keys]).find(
+    // Using `.includes()` will result a type error
+    // eslint-disable-next-line unicorn/prefer-includes
+    (key) => !logtoConfigKeys.some((element) => element === key)
+  );
+
+  if (invalidKey) {
+    log.error(
+      `Invalid config key ${chalk.red(invalidKey)} found, expected one of ${validKeysDisplay}`
+    );
+  }
+};
+
+export const getKey: CommandModule<unknown, { key: string; keys: string[] }> = {
```

Comment:
nit: getByKey, setByKey?

Reply:
good call. i think it should rename to `get-config`?

Reply:
LOG-4324